tests/securedrop/install.pm

```diff
@@ -0,0 +1,51 @@
+# The Qubes OS Project, https://www.qubes-os.org/
+#
+# Copyright (C) 2024 Francisco Rocha <francisco@freedom.press>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use base "installedtest";
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_gui_console;
+
+    assert_and_click("menu");
+    if (check_screen("menu-tab-favorites-active", 30)) {
+        # switch to apps tab
+        click_lastmatch();
+    }
+    assert_and_click("menu-vm-work");
+    assert_and_click("menu-vm-xterm");
+
+
+    assert_script_run('gpg --keyserver hkps://keys.openpgp.org --recv-key "2359 E653 8C06 13E6 5295 5E6C 188E DD3B 7B22 E6A3"');
```

Comment:
`assert_script_run` depends on seeing serial console output - serial console from "work" VM isn't directly connected to the one of the host; for this to work you either need to run something like `tail -F /var/log/xen/console/guest-work.log >> /dev/hvc0` in dom0 (we do that [here](https://github.com/QubesOS/openqa-tests-qubesos/blob/main/tests/guivm_startup.pm#L43)), or do all that from dom0's terminal via qvm-run

Reply:
Yes, that would work, but your test wouldn't detect if any of those command fails (other than possible some later step dom0 in dom0 failing).